### PR TITLE
adds onvisibilitychange to Document

### PR DESCRIPTION
**Pull request by DCtheTall**
for TypeScript issue [#21353](https://github.com/Microsoft/TypeScript/issues/21353)

### CODE REVIEW

File: inputfiles/addedTypes.json [hunk 1]
```diff
@@ -2713,5 +2713,11 @@
                 "type": "boolean"
             }
         ]
+    },
+    {
+        "kind": "property",
+        "interface": "Document",
+        "name": "onvisibilitychange",
+        "type": "(this: Document, ev: Event) => any"
```
Comment (HolgerJeromin) [inline inputfiles/addedTypes.json]: really `any` or better `void`?

Comment (DCtheTall) [reply inputfiles/addedTypes.json]: @HolgerJeromin I just followed the pattern I saw other event listeners on the Document type, it appeared the other type definitions had `any` as the return type.

If you think `void` is more appropriate happy to make that change.

Comment (HolgerJeromin) [reply inputfiles/addedTypes.json]: Oh, i have not checked that. I would keep it consistent, too :)

**Comment by DCtheTall** [comment]
@mhegazy I have yet to add support for adding an event listener for `visibilitychange` by using `addEventListener`. Going to try my hand at allowing the `DocumentEventMap` to be modified, I will open another PR if I get something working.

**Pull request merged**